stagemonitor/src/main/java/com/cerner/beadledom/stagemonitor/request/LogJsonRequestTraceReporter.java

```diff
@@ -35,16 +31,20 @@
    */
   public LogJsonRequestTraceReporter(ObjectMapper objectMapper) {
     this.logger = LoggerFactory.getLogger(LogJsonRequestTraceReporter.class);
+    this.errorLogger =
+        LoggerFactory.getLogger(LogJsonRequestTraceReporter.class.getSimpleName() + ".Error");
     this.objectMapper = objectMapper;
     this.requestMonitorPlugin = Stagemonitor.getPlugin(RequestMonitorPlugin.class);
     this.jsonRequestTraceLoggerPlugin = Stagemonitor.getPlugin(JsonRequestTraceLoggerPlugin.class);
   }
 
   //@VisibleForTesting
   LogJsonRequestTraceReporter(
```

Comment:
Correct.

This class only gets created here: https://github.com/cerner/beadledom/blob/master/stagemonitor/src/main/java/com/cerner/beadledom/stagemonitor/LogJsonRequestTraceReporterProvider.java

And used here (to register with stagemonitor): https://github.com/cerner/beadledom/blob/master/stagemonitor/src/main/java/com/cerner/beadledom/stagemonitor/RequestMonitorLifecycle.java